dev_guide/routes.adoc

```diff
@@ -188,8 +186,57 @@ spec:
   tls:
     termination: edge
 ----
-====
 
+For {product-title} Pro, you can use your own certificate and key files from a
+CA. You can omit the certificate and key files to use the default certificate.
+However, you can still omit the certificate and key files if you want to use the
+default certificate. With {product-title} Starter, you cannot specify a
```

Comment:
Whoops, these two sentences are redundant:

"You can omit the certificate and key files to use the default certificate."

"However, you can still omit the certificate and key files if you want to use the default certificate."

I'd delete the first and keep the latter (with "However" and "still").